python/lsst/ts/m2gui/utility_monitor.py

```diff
@@ -0,0 +1,439 @@
+            "J2-W10-3": False,
+            "J3-W11-1": False,
+            "J3-W11-2": False,
+            "J3-W11-3": False,
+            "J1-W12-1": False,
+            "J1-W12-2": False,
+            "J2-W13-1": False,
+            "J2-W13-2": False,
+            "J3-W14-1": False,
+            "J3-W14-2": False,
+        }
+
+        self.temperatures = {
+            "Intake-1": 0,
+            "Intake-2": 0,
+            "Exhaust-1": 0,
+            "Exhaust-2": 0,
+            "LG2-1": 0,
+            "LG2-2": 0,
+            "LG2-3": 0,
+            "LG2-4": 0,
+            "LG3-1": 0,
+            "LG3-2": 0,
+            "LG3-3": 0,
+            "LG3-4": 0,
+            "LG4-1": 0,
+            "LG4-2": 0,
+            "LG4-3": 0,
+            "LG4-4": 0,
+        }
+
+        self.displacements = {
+            "A1-Theta_z": 0,
+            "A2-Theta_z": 0,
+            "A3-Theta_z": 0,
+            "A4-Theta_z": 0,
+            "A5-Theta_z": 0,
+            "A6-Theta_z": 0,
+            "A1-Delta_z": 0,
+            "A2-Delta_z": 0,
+            "A3-Delta_z": 0,
+            "A4-Delta_z": 0,
+            "A5-Delta_z": 0,
+            "A6-Delta_z": 0,
+        }
+
+    def report_utility_status(self):
+        """Report the utility status."""
+
+        self.signal_utility.power_motor.emit(tuple(self.power_motor.values()))
+        self.signal_utility.power_communication.emit(
+            tuple(self.power_communication.values())
+        )
+
+        self.signal_utility.inclinometer.emit(self.inclinometer_angle)
+
+        for name, status in self.breakers.items():
+            self.signal_utility.breaker_status.emit((name, status))
+
+        self._report_temperatures()
+
+        self._report_displacements()
+
+    def _report_temperatures(self):
+        """Report the temperatures."""
+
+        for temperature_group in TemperatureGroup:
+            sensors = self.get_temperature_sensors(temperature_group)
+
+            temperatures = list()
+            for sensor in sensors:
+                temperatures.append(self.temperatures[sensor])
+
+            self.signal_utility.temperatures.emit((temperature_group, temperatures))
+
+    def get_temperature_sensors(self, temperature_group):
+        """Get the temperature sensors in a specific group.
+
+        Parameters
+        ----------
+        temperature_group : enum `TemperatureGroup`
+            Temperature group.
+
+        Returns
+        -------
+        `list [str]`
+            List of the sensors.
+        """
+
+        sensors = list()
+        for sensor in list(self.temperatures.keys()):
+            if sensor.startswith(temperature_group.name):
+                sensors.append(sensor)
+
+        return sensors
+
+    def _report_displacements(self):
+        """Report the displacements."""
+
+        for direction in DisplacementSensorDirection:
+            sensors = self.get_displacement_sensors(direction)
+
+            displacements = list()
```

Comment:
This can be shortened as:
```python
displacements = [self.displacements[sensor] for sensor in sensors]
```

Reply:
Fixed in the commit: [4d02847](https://github.com/lsst-ts/ts_m2gui/pull/7/commits/4d028476f7478675d575228b95db083bc1ae6e33). Thanks!